packages/btc/tests/Transaction.test.ts

```diff
@@ -471,6 +580,47 @@ describe('Transaction', () => {
       // const res = await service.sendBtcTransaction(tx.toHex());
       // console.log(`explorer: https://mempool.space/testnet/tx/${res.txid}`);
     });
+    it('Transfer protected UTXO, sum(ins) > sum(outs), change < fee', async () => {
+      const { builder, feeRate } = await createSendUtxosBuilder({
+        from: accounts.charlie.p2wpkh.address,
+        inputs: [
+          {
+            txid: '4e1e9f8ff4bf245793c05bf2da58bff812c332a296d93c6935fbc980d906e567',
+            vout: 1,
+            value: 2000,
+            addressType: AddressType.P2WPKH,
+            address: accounts.charlie.p2wpkh.address,
+            scriptPk: accounts.charlie.p2wpkh.scriptPubkey.toString('hex'),
+          },
+        ],
+        outputs: [
+          {
+            address: accounts.charlie.p2wpkh.address,
+            value: 1000,
+            protected: true,
+          },
+        ],
+        feeRate: 10,
+        source,
+      });
+
+      // Sign & finalize inputs
+      const psbt = builder.toPsbt();
+      psbt.signAllInputs(accounts.charlie.keyPair);
+      psbt.finalizeAllInputs();
+
+      expect(psbt.txInputs.length).toBeGreaterThanOrEqual(2);
+      expect(psbt.txOutputs).toHaveLength(1);
```

Comment:
If there are two or more inputs, is the change output required?

Reply:
In this test, the change is designed to be returned in outs[0] because it is a `protected output`. A protected output cannot be removed from the outs, but the free value (free value = output.value - minUtxoSatoshi) defined in the output can be deducted or added.

Reply:
Make sense